spec/realtime/message.test.js

```diff
@@ -690,6 +690,34 @@ define(['ably', 'shared_helper', 'async'], function(Ably, helper, async) {
 			});
 	};
 
+	/* For now realtime doesn't yet support the extras field, so stub transport.send to test it */
```

Comment:
@SimonWoolf I would really like to see the spec codes in all tests if possible.  We do this now for iOS and Android, and I've done it for all new Ruby tests and retrospectively went back and added a few for existing tests when I touched those files / sections.  It makes it so much easier to figure out if a test exists for a test spec item, and later when specs change, it's easy again to search, find the test, and amend if necessary.  I am not sure how much work it would be, but it would be good to a) update the tests in `0.9 branch` with the spec codes, b) make sure the `ably-js` spreadsheet is up to date.  The reported test coverage still feels quite low, but expect that is not accurate?